mod-source-record-storage-server/src/main/java/org/folio/dao/RecordDaoImpl.java

```diff
@@ -390,6 +406,18 @@ private Condition getMatchedFieldCondition(MatchField matchedField, String parti
     return condition(sql);
   }
 
+  private static String getComparisonValue(Filter.ComparisonPartType comparisonPartType) {
```

Comment:
Thanks @VRohach , I agree with using the switch statement, but I don't agree with separating the lines because, in my opinion, it doesn't make the code more understandable.

Reply:
@Aliaksandr-Fedasiuk  Got it. Thanks. Then left the best approach from your side.